src/main/java/amidst/gui/profileselect/LocalProfileComponent.java

```diff
@@ -81,6 +82,13 @@ private boolean tryLoad() {
 		try {
 			Log.i("using minecraft launcher profile '" + getProfileName() + "' with versionId '" + getVersionName()
 					+ "'");
+					
+			String possibleModProfiles = "optifine|forge";
+            if (Pattern.matches(possibleModProfiles, getProfileName().toLowerCase())) {
+                Log.e("Amidst does not support modded Profiles, please select or create an unmodded profile");
+                return false;
+            }
```

Comment:
The if-block uses spaces instead of tabs for its indentation. Please use Eclipse and import the preference file that is included in this repository to ensure that your commits adhere to our coding style.